app/src/main/java/com/example/workout_timer/models/RestTimer.java

```diff
@@ -0,0 +1,35 @@
+package com.example.workout_timer.models;
```

Comment:
These two Timers don't do anything yet. I plan on updating them when I do the TimerActivity